packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -149,6 +151,7 @@ class CollapsibleMenuViewItem extends React.Component {
               isReversed={isReversed}
               isDisabled={isDisabled}
               onClick={this.handleButtonClick}
+              isIconOnly={faceUpText.length === 0}
```

Comment:
Is there a difference between using the `faceUpText` length and the original `isIconOnly` prop?

Reply:
When I made this change, I thought there was a scenario where I had to use the `faceUpText` length and no just `isIconOnly` but I can't remember or recreate what it was. I switched back to use `isIconOnly` and there is no extra padding on the collapsible menu toggle. https://github.com/cerner/terra-core/pull/1206/commits/c0f8bc677477b692dd3f318e73bf4a7937de70dd